src/Logger.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\Component;
+
+use Monolog\Formatter\LineFormatter;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger as MonologLogger;
+
+class Logger extends MonologLogger
+{
+    public function __construct(
+        ?StreamHandler $errorHandler,
+        ?StreamHandler $logHandler
+    ) {
+        parent::__construct('php-component');
+
+        if ($errorHandler === null) {
+            $errorHandler = new StreamHandler('php://stderr');
+        }
+        $errorHandler->setBubble(false);
+        $errorHandler->setLevel(MonologLogger::ERROR);
```

Comment:
dal bych sem WARN

Reply:
Proč? Byla tam [v DB writeru dokonce NOTICE](https://github.com/keboola/google-sheets-writer/blob/master/src/Keboola/GoogleSheetsWriter/Logger/HandlerFactory.php#L26)! Ale nepřijde mi to správně. Pokud je to error log, tak by tam měli být chyby, ne? Warning je např. "no columns specified, using autodetection" (což je něco co by mohlo být žluté, ale nemělo by být červené, protože to není chyba), ne?

Reply:
OK. Dává to asi smysl.